docs/eventing/samples/kubernetes-event-source/README.md

```diff
@@ -38,16 +74,20 @@ all the YAML files deployed in this sample to point at that namespace.
 
 1. Create a Service Account that the `ApiServerSource` runs as. The
    `ApiServerSource` watches for Kubernetes events and forwards them to the
-   Knative Eventing Broker. Create a file named `serviceaccount.yaml` and copy
-   the code block below into it.
+   Knative Eventing Broker.
+
+   {{< tabs name="create-service-account" default="By YAML" >}}
+   {{% tab name="By YAML" %}} Use following command to create the service
```

Comment:
```suggestion
   {{% tab name="By YAML" %}} Use the following command to create the service
```

Reply:
Added "the" so that it matches some of previous documentation. Not a big change, it reads a little bit better, but might not really be a required change.